source/models/config.py

```
@@ -0,0 +1,159 @@
+    """Handles configuration loading, saving, and merging"""
+    
+    @staticmethod
+    def load():
+        """Load configuration from file"""
+        if not path.exists(CONFIG_PATH):
+            return DEFAULT_CONFIG.copy()
+        
+        try:
+            with open(CONFIG_PATH, "r") as f:
+                return json.load(f)
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)